lochness/__init__.py

```diff
@@ -378,10 +378,10 @@ def listdir(Lochness, d):
 
 def attempt(f, Lochness, *args, **kwargs):
     '''attempt a function call'''
-    if len(attempt.warnings) >= 5:
-        # lochness.email.attempts_error(Lochness, attempt)
-        attempt.warnings = []
-        raise AttemptsError('too many attempt warnings')
+    # if len(attempt.warnings) >= 5:
+        # # lochness.email.attempts_error(Lochness, attempt)
+        # attempt.warnings = []
+        # raise AttemptsError('too many attempt warnings')
```

Comment:
First, will removal of this block allow infinite attempts?

Second, we should get rid of dead codes. Besides, we can put all the `#` along one vertical line for neatness.

Reply:
The attempt functionality could be very useful, but for our situation where `mindlamp` and `upenn` modalities are not fully working yet, I have left them commented out for most efficient testing. I would like to vote for removing `attempts` at least until we have stable dataflow. Let me know what you think. We could then think about totally removing this part of dead codes.

Reply:
Can this be the way to retain the comments for now?

```
    '''
    if len(attempt.warnings) >= 5:
        lochness.email.attempts_error(Lochness, attempt)
        attempt.warnings = []
        raise AttemptsError('too many attempt warnings')
    '''
```

Reply:
9f99fa2